### PR TITLE
Add use_no_compression option

### PR DESCRIPTION
**Pull request by emcrisostomo**
`use_no_compression` configures the fetch and the extract phases to deal with a non-compressed file with no extract extension.

When creating the port for Amazon AWS `ecs-cli`, I had to deal with the case in which the port has to download an uncompressed binary file (see https://github.com/aws/amazon-ecs-cli).

The port (submitted in https://github.com/macports/macports-ports/pull/1089), works with the following configuration:

```
extract.suffix
extract.cmd             cp
extract.pre_args
extract.post_args       ${worksrcpath}
extract.mkdir           yes
```

Basically, the extract suffix is set to an empty string and the downloaded file is just copied into `${worksrcpath}`, which gets created by setting `extract.mkdir` to `yes`.

I thought it may be useful to deal with this case natively, and I added the `use_no_compression` option.

### CODE REVIEW

**Comment by emcrisostomo** [comment]
After learning a much simpler workaround, I realise this is probably an over-engineered solution for a problem that's easy to overcome. Closing.

**Comment by ryandesign** [comment]
This is a pretty uncommon situation that I don't think needs special treatment in MacPorts base.

**Comment by emcrisostomo** [comment]
> This is a pretty uncommon situation that I don't think needs special treatment in MacPorts base.

I agree.
